bin/sequenceserver

```diff
@@ -245,6 +245,8 @@ MSG
           puts
           puts 'Installing NCBI BLAST+.'
           puts "RUBY_PLATFORM #{RUBY_PLATFORM}"
+          puts 'Archive will be extracted in ~/.sequenceserver directory.'
```

Comment:
There's ambiguity in 'Archive will be extracted ...' msg too. Nonetheless, this definitely improves on what we have, so I'm merging the PR.